src/frontend/components/Felleskomponenter/BlokkerTilbakeKnappModal/BlokkerTilbakeKnappModal.test.tsx

```diff
@@ -43,9 +43,6 @@ const fraPdlSomIBarnMedISøknad: Partial<IBarnMedISøknad> = {
 };
 
 describe('Ingen navigering tilbake til søknad fra kvitteringssiden', () => {
-    beforeEach(() => {
```

Comment:
Prøver å fjerne så mye mocking som jeg kan, men har ikke fjernet alt. Et sted må grensen gå for denne PRen. Vi ønsker å gå vekk fra mocking i så stor grad som mulig, og heller lene oss på MSW for å gi tilbake riktige responser.